apps/fishing-map/pages/index.tsx

```diff
@@ -14,11 +16,51 @@ const AppNoSSRComponent = dynamic(() => import('../features/app/App'), {
   ssr: false,
 })
 
-const Index = () => {
+const HeadDescriptionByLocation = ({ isMarineManagerLocation }) => {
   return (
-    <MapProvider>
-      <AppNoSSRComponent />
-    </MapProvider>
+    <Head>
+      <meta
+        property="og:description"
```

Comment:
There are already meta tags on the _document.tsx file https://github.com/GlobalFishingWatch/frontend/blob/22a402c5393346f456e01e3e81022880c2d77a7b/apps/fishing-map/pages/_document.tsx#L16

Reply:
Based on [this documentation](https://nextjs.org/docs/api-reference/next/head) `In this case only the second <meta property="og:title" /> is rendered. meta tags with duplicate key attributes are automatically handled.` I don't think that will be a problem.
And we need to handle it at index.tsx page as document can't read the originalUrl